app/src/main/java/org/metabrainz/android/presentation/features/search/SearchPagingSource.kt

```diff
@@ -19,6 +20,7 @@ class SearchPagingSource(val mbEntityType: MBEntityType, val query: String) : Pa
         return try {
             val response = service.searchEntity(mbEntityType.entity, query, pageSize, offset)?.string()
             var count = LoadResult.Page.COUNT_UNDEFINED
+            val entity = JsonParser.parseString(response).asJsonObject.get(mbEntityType.entity+"s").asJsonArray
```

Comment:
So I did some checking and I found we can check every entity like this. I had to append an extra "s" because what we are getting in response is having an extra "s" in the entity, and now we can check each entity correctly.
Sir, please let me know if it could help or do I need to make some other changes.

Reply:
The variable name being `entity` is odd here and doesn't justify its usage. Also, can't we just declare the data being as a variable from line 40 and check whether it is empty or not for this case?